### PR TITLE
add photo quality setting to mergin project properties

### PR DESCRIPTION
**Pull request by alexbruy**
Adds dropdown to Mergin project settings dialog with the following options: "Original", "High", "Medium" and "Low". By default "Original" setting is used to keep backward compatibility.

зображення

Refs lutraconsulting/input/issues/841.

### CODE REVIEW

**Review by PeterPetrik — APPROVED** [review]

**Comment by wonder-sk** [comment]
May I suggest that this new option will also get a dedicated group box, and a short explanatory label, e.g.: "This option is used by Input when adding pictures to the project (taken from camera or using existing picture from gallery)."

File: Mergin/project_settings_widget.py [hunk 1]
```diff
@@ -28,6 +29,16 @@ class ProjectConfigWidget(ProjectConfigUiWidget, QgsOptionsPageWidget):
     def __init__(self, parent=None):
         QgsOptionsPageWidget.__init__(self, parent)
         self.setupUi(self)
+
+        self.cmb_photo_quality.addItem("Original", 0)
+        self.cmb_photo_quality.addItem("High", 1)
+        self.cmb_photo_quality.addItem("Medium", 2)
+        self.cmb_photo_quality.addItem("Low", 3)
```
Comment (wonder-sk) [inline Mergin/project_settings_widget.py:36]: can we get some extra info on what the High/Medium/Low mean? either expected size in MB or expected size in pixels (or both)

Comment (alexbruy) [reply Mergin/project_settings_widget.py]: Added information about approximate size, as I'm not sure about correct wording for pixel size. We have to take into account image orientation so it will be either width or height

**Comment by PeterPetrik** [comment]
It wooudl be good to add expanatory text as Martin suggested

**Comment by alexbruy** [comment]
Explanatory text added. Also added link to the selective sync documentation.

зображення

**Comment by tomasMizera** [comment]
I suggest rewording the text for selective sync to "With selective sync enabled Input will not download photos from other surveyors to make synchronisation faster. Check out more in the *documentation*"

Otherwise, I believe we are ready to merge (afaik it passed testing by @jozef-budac ). @wonder-sk

**Comment by wonder-sk** [comment]
Good suggestion @tomasMizera - let's update the text :-)
